backend/fms_core/services/dataset.py

```diff
@@ -276,8 +277,12 @@ def ingest_run_validation_report(report_json):
         readset_by_name[readset_name] = readset_obj
         for key in readset:
             if key in ACCEPTED_DATASET_FILE_TYPES and readset[key]:
+                file: DatasetFileReport = readset[key]
+                if file.get('final_path') is None:
+                    warnings.append(("final_path not provided for file type {0} of readset {1}", key, readset_name))
```

Comment:
The path is required (a file without path is nothing) this should raise an error.